android/samples/sample-gltf-viewer/src/main/AndroidManifest.xml

```diff
@@ -9,7 +9,9 @@
       android:roundIcon="@mipmap/ic_launcher_round"
       android:supportsRtl="true"
       android:theme="@style/AppTheme">
-    <activity android:name="com.google.android.filament.gltf.MainActivity" android:screenOrientation="portrait">
+    <activity android:name="com.google.android.filament.gltf.MainActivity"
+        android:screenOrientation="fullSensor"
```

Comment:
This attribute is not needed, you can just remove it, it's the default behavior.

Reply:
Hm, the viewport stops rotating if I leave screenOrientation to its default (which is `unspecified`). I'm using an emulator if that matters.

Btw, the `configChanges` attribute is working perfectly; without it, the model reloads from scratch. Thanks for your help with that one.